src/bench/addrman.cpp

```diff
@@ -82,7 +82,7 @@ static void AddrManAdd(benchmark::Bench& bench)
 
 static void AddrManSelect(benchmark::Bench& bench)
 {
-    CAddrMan addrman;
+   CAddrMan addrman(/* deterministic */ false, /* consistency_checks */ 0);
```

Comment:
nit: here and below - indentation of 3 spaces

Reply:
oops. Fixed.